__tests__/api.test.js

```diff
@@ -289,6 +289,50 @@ describe("/api/reviews/:review_id/comments", () => {
   });
 });
 
+describe("GET /api/users", () => {
+  test("GET /api/users responds with status 200 and an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((result) => {
+        expect(Array.isArray(result.body)).toBe(true);
+        expect(result.body.length).toBe(4);
+      });
+  });
+  test("GET /api/users responds with an array of user objects with the required properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((result) => {
+        result.body.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
+        });
+      });
+  });
+  test("GET /api/users responds with an array of review objects with properties which are of the correct data type", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((result) => {
+        result.body.forEach((review) => {
+          expect(typeof review.username).toBe("string");
+          expect(typeof review.name).toBe("string");
+          expect(typeof review.avatar_url).toBe("string");
+        });
+      });
+  });
+  test("GET /api/users responds with status 404 and error message if endpoint is incorrect", () => {
```

Comment:
This should be covered by your existing invalid path test, so you can remove this if you want